### ISSUE TITLE
Settings are extremely touch-unfriendly

### ISSUE THREAD

**Issue by ildar** [gerstrong/Commander-Genius#241]
The game settings seem to have two "active item"s (cursors): one for keyboard another for touch. This is very misguiding. Please, fix.

**Comment by gerstrong** [comment]
Actually if we get SDL2 Android Pelya support I might change that. Have been working on that. Need more time to do so...

**Comment by ildar** [comment]
Does that mean that the Android port is mainly developed by @pelya and I
should file bugs at https://github.com/pelya/commandergenius ?

**Comment by gerstrong** [comment]
Yeah, it is but he has been busy with other stuff. SDL2 is not a mayor priority right now... SDL 1.2 works fine for us, so far...
